DataFormats/METReco/src/classes_def.xml

```diff
@@ -172,6 +172,7 @@
   </class>
   <class name="edm::Wrapper<reco::HcalHaloData>"/>
 
+  <class name="HaloTowerStrip"/>
```

Comment:
Sorry, wasn't sure if the struct needed a class version. I've added it along with a checksum. Thanks!